my-dsl.md

```diff
@@ -1,39 +1,54 @@
 # Language
 
-_What is the name of the language? Link the name to its webpage
-(if appropriate)._
+_What is the name of the language? Link the name to its webpage (if appropriate)._
+
+The DSL that I chose is Regualr Languages or Regex Expressions. The reason why I chose this DSL is because I have worked with them before in a prior internship. [java.util.regex regex link for java](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)
```

Comment:
Regex is very cool, I feel like I've never used it enough to fully understand it.